projects/storefrontstyles/scss/components/content/media/_media.scss

```diff
@@ -1,7 +1,7 @@
 %cx-media {
   position: relative;
-  display: flex;
-  align-items: stretch;
+  display: block;
```

Comment:
is this needed? I'd rather not be opinionated about the display mode, can we do that.

Reply:
All of these changes are in develop back in may. We are trying to get this back ported into 2.0.x to fix the stretchy images. So if there needs to be changes that would have to be done in another ticket

Reply:
@tobi-or-not-tobi we're only backporting what was missed from `develop` since May, but I'll take note of your comments for later improvements.